### ISSUE TITLE
Images are cut off on page

### ISSUE THREAD

**Issue by diantekyrie** [ampproject/amphtml#21937]
## What's the issue?
Images are cut off on page

Briefly describe the bug.
Images are not fully displayed.

## How do we reproduce the issue?

1. Go to https://m.aliexpress.com/category/100003070.html?spm=a2g0n.home.category.2
2. Open developer tools
3. Select a category from the selection 

## What browsers are affected?
macOSX Chrome Version 73.0.3683.103 (Official Build) (64-bit)
macOSX Safari Version 12.0.3 (14606.4.5)
macOSX FireFox Verison 66.0.3 (64-bit)
iOS Safari
iOS Chrome: Version 73.0.3683.68

## Which AMP version is affected?

Dev Channel Version: 1904200955460
Stable Version: 1904091426070

Note: On mobile device you can swipe to completely see images.

Browser Image:
![Screen Shot 2019-04-22 at 3 15 14 PM](https://user-images.githubusercontent.com/48487406/56535818-75fee800-6511-11e9-9677-6d8cd3a67158.png)

Mobile Image:
![image1](https://user-images.githubusercontent.com/48487406/56536018-e86fc800-6511-11e9-9583-849714f895ee.png)

**Comment by aghassemi** [comment]
Thanks @diantekyrie.

@leonalicious Looks like an implementation bug. 
The category page properly has a 
```
.category-page {
    padding-top: 56px;
```
padding to account for the header but sub-category page seems to be missing it.

**Comment by aghassemi** [comment]
I am closing this issue on our side as the issue is on the implementation side.
